CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [v6.8.0] 2021-03-16
```

Comment:
This should be added when we release the version, or please keep in mind to update the date if you intend to do it in once.

Reply:
Sure.